eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -59,7 +59,8 @@
           "SupportsRecording": true,
           "CollectCoverage": true
         }
-      }
+      },
+      "AdditionalTestArguments": "/p:UseProjectReferenceToAzureClients=false"
```

Comment:
Did we intentionally want to add this to the include?

Reply:
I did. It's so that this value is present on the `Include` value added to the matrix at the end.